components/map/Clusters.js

```diff
@@ -0,0 +1,96 @@
+import { useState } from "react";
+import { Marker, useMap } from "react-leaflet";
+import useSupercluster from "use-supercluster";
+import UserMarker from "./UserMarker";
+import styles from "./Clusters.module.css";
+
+export default function Clusters({users}) {
+  const map = useMap();
+  const mapB = map.getBounds();
+  const [bounds, setBounds] = useState([
+    mapB.getSouthWest().lng,
+    mapB.getSouthWest().lat,
+    mapB.getNorthEast().lng,
+    mapB.getNorthEast().lat
+  ]);
+  const [zoom, setZoom] = useState(map.getZoom());
+
+  function updateMap() {
+    const b = map.getBounds();
+    setBounds([
+      b.getSouthWest().lng,
+      b.getSouthWest().lat,
+      b.getNorthEast().lng,
+      b.getNorthEast().lat
+    ]);
+    setZoom(map.getZoom());
+  }
+
+  map.on('moveend', function() {
+    updateMap();
+  })
+
+  const { clusters, supercluster } = useSupercluster({
+    points: users,
+    bounds,
+    zoom,
+    options: { radius: 50, maxZoom: 18 }
+  });
+
+  const icons = {};
+  const fetchIcon = (count) => {
+    const size =
+      count < 25 ? 'small' :
+      count < 100 ? 'medium' : 'large';
```

Comment:
@dan-mba is this where we are deciding how big/small each cluster is?  I noticed there's more overlapping with singular icon images and clusters at the scale of the full map. I don't think this affects UX so is likely fine for now, but if anyone reports any issues with the overlap we will now have, I was curious if we have control over the cluster "capture area"

Reply:
@amandamartin-dev the size does not change the marker size, it only adjusts the color scheme.